CentralHub.Api/Controllers/TrackerController.cs

```diff
@@ -50,4 +50,11 @@ public async Task RemoveTracker(Tracker tracker, CancellationToken cancellationT
     {
         await _roomRepository.RemoveTrackerAsync(tracker, cancellationToken);
     }
+
+    [HttpPost("measurement")]
+    public async Task PostMeasurement(int id, List<Measurement> measurements, CancellationToken token)
+    {
+        _localizationTargetService.AddMeasurementsAsync(id, measurements, token);
+        await Task.CompletedTask;
```

Comment:
Why are we awaiting a completed task?
This does nothing remove this line
Also await the call to AddMeasurementsAsync

Reply:
Wait no its not async.